manifests/plugin/rabbitmq.pp

```diff
@@ -61,7 +61,7 @@
 ) {
   include collectd
 
-  if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'],'8') >= 0 {
+  if ($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8') or ($facts['os']['name'] == 'Ubuntu' and $facts['os']['release']['major'] == '20.04') {
```

Comment:
This means it no longer works for EL9 nor Fedora. While not technically supported, I think it's good to look to the future.

Reply:
good catch, fixed it